imagekit-sdk/build.gradle

```diff
@@ -13,7 +13,12 @@ repositories {
 }
 
 dependencies {
-    compile fileTree(include: ['*.jar'], dir: '../libs')
+    compile 'commons-codec:commons-codec:1.13'
+    implementation 'com.squareup.retrofit2:converter-gson:2.3.0'
+    implementation 'com.google.code.gson:gson:2.7'
+    implementation 'com.squareup.okhttp3:okhttp:3.10.0'
```

Comment:
What happens if someone is already using another version of okhttp3?

Reply:
gradle picks the higher version:
https://docs.gradle.org/current/userguide/dependency_resolution.html#sec:version-conflict